velo:
  point_o:
    x:
    y:
    z:
  point_z:
    x:
    y:
    z:
  point_y:
    x:
    y:
    z:
rs:
  point_o:
    x:
    y:
    z:
  point_z:
    x:
    y:
    z:
  point_y:
    x:
    y:
    z: